test/test-apimatic-core/union_type_tests/one_of_test.rb

```diff
@@ -133,10 +133,9 @@ def test_deserialize_nil_morning_type_one_of
     deserialized_morning = ApiHelper.json_deserialize(_morning_json)
     _one_of = _one_of.validate(deserialized_morning)
     actual_morning = _one_of.deserialize(nil)
-    expected_morning = nil
 
     assert _one_of.is_valid
-    assert_equal(expected_morning, actual_morning, 'Actual did not match the expected.')
+    assert_nil actual_morning
```

Comment:
same thing mentioned for anyof tests is applicable here

Reply:
It is a refactoring that we can do against the testing framework that we are using currently which is minitest.